src/EnergyPlus/HeatingCoils.cc

```diff
@@ -304,8 +314,6 @@ namespace HeatingCoils {
         bool DXCoilErrFlag; // Used in GetDXCoil mining functions
         bool errFlag;
 
-        auto &HeatingCoil(state.dataHeatingCoils->HeatingCoil);
```

Comment:
I was confused at first, but yes, great, we should definitely eliminate references to arrays.